lib/middleware/token.js

```diff
@@ -51,6 +51,8 @@ module.exports = function token(server, options) {
   if (!server) { throw new TypeError('oauth2orize.token middleware requires a server argument'); }
   
   return function token(req, res, next) {
+    if (!req.body) { return next(new Error('OAuth2orize requires body parsing. Did you forget app.use(express.bodyParser())?')); }
```

Comment:
express does not bundle bodyParser any more. See http://expressjs.com/4x/api.html#req.body for details.